stackslib/src/chainstate/stacks/block.rs

```diff
@@ -599,6 +600,10 @@ impl StacksBlock {
                     return false;
                 }
             }
+            if !tx.auth.is_supported_in_epoch(epoch_id) {
+                error!("Order independent multisig transactions not supported before Stacks 3.0");
```

Comment:
Tweak -- this error messasge should be more generic (e.g. "this authentication mode is not supported in {epoc_hid}")